client/src/App.tsx

```diff
@@ -1,7 +1,34 @@
+import { useEffect } from 'react';
+import { Navigate, Route, Routes } from 'react-router-dom';
 import './App.scss';
+import { Header } from './components/Header/Header';
+import { useAppDispatch } from './hooks/redux';
+import { Home } from './pages/Home/Home';
+import { SignIn } from './pages/SignIn/SignIn';
```

Comment:
Proposition: You can create index.ts file in the pages, and reexport every page from it, and than you will be able to do 
`import { Home, SignIn, SignUp } from './pages';`